internal/iam/command_rbac.go

```diff
@@ -19,6 +19,7 @@ var (
 	identityNamespace       = optional.NewString("identity")
 	flinkNamespace          = optional.NewString("flink")
 	workloadNamespace       = optional.NewString("workload")
+	flinkModelNamespace     = optional.NewString("flinkmodel")
```

Comment:
This is clearly not scaling very well since your team has had to open 9 separate PRs to gradually add all of these namespaces... Has your team considered controlling this list of namespaces in the backend?

Reply:
cc @AydenAdair

Reply:
We can consider looking into controlling this via the backend but often the CLI and UI have different needs (ie. these roles only need to be available in the CLI for the EA) so doing it via the backend may end up being more work. These roles are scheduled to be in EA this week though so we may not have enough time to figure that out in this change.

Reply:
Let's merge this PR as is, but in the future it would be great to add this to the backend. I assume you could add a switch to distinguish if a request is coming from the UI vs CLI in the backend!

Reply:
(At the very least, let's create a ticket for this!)